content/en/blog/_posts/2022-08-15-immutability-with-cel.md

```diff
@@ -0,0 +1,526 @@
+---
+layout: blog
+title: "Enforce CRD Immutability with CEL Transition Rules"
+date: 2022-08-15
+slug: enforce-immutability-using-cel
+canonicalUrl: https://www.kubernetes.dev/blog/2022/08/15/enforce-immutability-using-cel
+---
+
+**Author:** [Alexander Zielenski (Google)](https://github.com/alexzielenski)
+
+---
+
+# Introduction
+Immutable fields can be found in a few places in the built-in Kubernetes types such as PodSpec, EphemeralContainers, Finalizers, and more.
```

Comment:
```suggestion
Immutable fields can be found in a few places in the built-in Kubernetes types.
For example, you can't change the `.metadata.name` of an object. Specific objects
have fields where changes to existing objects are constrained; for example, the
`.spec.selector` of a Deployment.
```

You _can_ mutate finalizers and ephemeral containers for an existing Pod, and to suggest otherwise is liable to confuse folk.